contact.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+            <ul id="navigation-bar"></ul>
+            <div>
+                <img id="header-img" src="gnysw2objzekbagrqiax.png" alt="Microverse">
+            </div>
+			<form action="index.html">
+			<ul>
+            <li><button type="button" class="nav-link">Home</button></li>
+			</ul>
+			</form>
+			<form action="contact.html">
+			<ul>
+            <li><button type="button" class="nav-link">Contact Us</button></li>
+			</ul>
+			</form>
+			<form action="review.html">
+			<ul>
+            <li><button type="button" class="nav-link">Reviews</button></li>
+            </ul>
+			</form>
+        </nav>
+    </header>
+    <form id="form">
+        <input type="email" id="email" name="emailid" placeholder="Enter your email"><br>
+        <input type="number" id="phonenumber" name="phoneid"><br>
+        <input type="text" id="nameclient" name="nameid"><br>
+        <input onclick="window.location.href='www.freecodecamp.com/email-submit';" id="submit" type="submit"
```

Comment:
Can you kindly modify the name tag which should be the same as the input type tag? Also, if possible, try to include the HTML tags in semantic style.